### PR TITLE
yamlvalidate passing versions of test-descriptors

### PR DESCRIPTION
**Pull request by midlakelands**
Updated schema to be properly parsed by yamlvalidate

### CODE REVIEW

File: test-descriptor/test-descriptor-schema.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  author:
+    description: "The person or organization that created the test descriptor."
+    type: "string"
+  description:
+    description: "A longer description of the test descriptor."
+    type: "string"
+  test_type:
+    description: "test type"
+    $ref: "#/definitions/testtype"
+  test_execution_file:
```
Comment (felipevicens) [inline test-descriptor/test-descriptor-schema.yml:46]: Can be multiple files?

File: test-descriptor/examples/test-descriptor-example.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        parameter_definition: "Docker instance for TTCN3 runtime engine"
+        parameter_value: "TTCN3.docker"
+        content_type: "application/octet-stream"
+test_category:
+    - category_type: performance   
+
+#test tags are 
+test_execution:
+    - test_tag: moongen
+      tag_id: tag_001
```
Comment (felipevicens) [inline test-descriptor/examples/test-descriptor-example.yml:27]: Should be test_id to match the schema proposed.

Comment (midlakelands) [reply test-descriptor/examples/test-descriptor-example.yml:27]: will modify schema to correct should be tag_id

File: test-descriptor/test-descriptor-schema.yml (hunk 1 above)
Comment (midlakelands) [reply test-descriptor/test-descriptor-schema.yml:46]: hmm maybe test_resources is a better and more generic name?

Comment (midlakelands) [reply test-descriptor/test-descriptor-schema.yml:46]: and yes can be multiple files

**Pull request closed without merging**